### PR TITLE
Allow Symfony 7

### PR DESCRIPTION
**Pull request by OskarStark**
Replaces
* #384

### CODE REVIEW

File: .github/workflows/ci-6.x.yml [hunk 1]
```diff
@@ -43,13 +43,12 @@ jobs:
     strategy:
       matrix:
         php:
-          - '8.0'
-          - '8.1'
```
Comment (OskarStark) [inline .github/workflows/ci-6.x.yml:47]: Symfony 7 requires PHP 8.2

File: composer.json [hunk 2]
```diff
@@ -11,18 +11,18 @@
         }
     ],
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.2",
```
Comment (OskarStark) [inline composer.json:14]: Symfony 7 requires PHP 8.2

**Comment by OskarStark** [comment]
> Problem 1
    - Root composer.json requires damienharper/auditor dev-master -> satisfiable by damienharper/auditor[dev-master].
    - damienharper/auditor dev-master requires symfony/cache ^5.4|^6.0 -> found symfony/cache[v5.4.0, ..., v5.4.32, v6.0.0, ..., v6.4.0] but these were not loaded, likely because it conflicts with another require.
Error: Your requirements could not be resolved to an installable set of packages.

**Comment by DamienHarper** [comment]
@OskarStark https://github.com/DamienHarper/auditor/pull/190 will fix this

**Review by DamienHarper — APPROVED** [review]

**Pull request merged**